packages/fxa-content-server/app/tests/spec/models/user.js

```diff
@@ -68,6 +71,13 @@ describe('models/user', function () {
     user = null;
   });
 
+  describe('initialize',function () {
```

Comment:
Writing a unit test for the first time...Does this look right? @shane-tomlinson

Reply:
This does!